README.md

```diff
@@ -77,22 +77,7 @@ See the system requirements for the [OpenTok Android SDK](https://tokbox.com/dev
    npx pod-install
    ```
 
-2. **For React Native versions prior to 0.60**:
-
-   * Add this to your Podfile:
-
-     ```
-     target '<YourProjectName>' do
-         # Pods for <YourProject>
-         pod 'OTXCFramework', '2.30.1'
-     end
-     ```
-   
-   * Run `react-native link opentok-react-native`.
-
-   These steps are not necessary in React Native version 0.60 and later.
-
```

Comment:
v2.31.0 only supports the React Native new architecture (which was not supported in React Native 0.59).